src/utils/axios.js

```diff
@@ -4,21 +4,30 @@ const api = axios.create({
   baseURL: "https://iz-nc-news.onrender.com/api",
 });
 
-
-const getArticles = api.get("/articles");
+const getArticles = function (queryString) {
+  return api.get(`/articles${queryString}`);
```

Comment:
Good work on using the API to filter the topics but don't forget we can use [axios params](https://notes.northcoders.com/courses/js-front-end/axios#axios-params) here.

Reply:
Will definitely look into this and the above comment to make it cleaner later on.